File: Java/Quiz/Contestant.java
import java.util.Scanner;

class Contestant {
	private int cid;
	private String name;
	private String dob;
	private char gender;

	public void newContestant() {
		Scanner scan = new Scanner(System.in);
		System.out.print("Enter contestant id: ");
		cid = scan.nextInt();
		System.out.print("Enter Name: ");
		name = scan.next();
		System.out.print("Enter date of birth: ");
		dob = scan.next();
		System.out.print("Enter gender: ");
		gender = scan.next().charAt(0);
	}
	public void editContestant(){
	
	}
	public void completeList(){
	
	}
}

File: CPP/Quiz/Contestant.cpp
#include<iostream>
using namespace std;
class contestant
{
	int cid;
	string name;
	string dob;
	char gender;
public:
	void new_contestant();
	void edit_contestant();
	void complete_list();
};

void contestant :: new_contestant()
{
	cout<<"Enter contestant id: ";
	cin>>cid;
	cout<<"Enter name: ";
	getchar();
	getline(cin,name);		//For reading multi-word strings
	cout<<"Enter date of birth: ";
	cin>>dob;
	cout<<"Enter gender: ";
	cin>>gender;
}
